steering/CONTRIBUTION-FORMULA.md

```diff
@@ -4,13 +4,12 @@ The [steering committee charter](CHARTER.md) states "Each year, when
 Contribution Seat terms begin, the Steering Committee will vote on an exact
 formula and procedure for determining allocation for the following year."
 
-For the February 2021 to January 2022 term, the contribution formula shall be
-**merged pull requests** attributed to a Contributing Company,
+For the February 2022 to January 2023 term, the contribution formula shall be
+**contributions** (review, comment, commit, issue or PR on GitHub) attributed to a Contributing Company,
 and the [D'Hondt method](https://en.wikipedia.org/wiki/D%27Hondt_method) of
 allocation of seats.
 
-Attribution is done using the company attribution table used by the
-istio-policy-bot. Requests for changes to attribution can be made by e-mail
-to elections@discuss.istio.io.
+Attribution is done using the [cncf/gitdm](https://github.com/cncf/gitdm) developer affiliation files.
+Changes to attribution can be made by pull request to that repository.
 
-[The method and results are published here](https://docs.google.com/spreadsheets/d/1Dt-h9s8G7Wyt4r16ZVqcmdWXDuCaPC0kPS21BuAfCL8/edit#gid=0).
+[The method and results for 2021-2022 are published here](https://docs.google.com/spreadsheets/d/187LDk96ggccHa_IBBGLEHYEBmR5eidsEQUDy2nE9kxo/edit).
```

Comment:
The tabs still show "PRs merged" as the criteria on the spread sheet

Reply:
That's correct - we used PRs merged for 21-22 (this proposal is for 22-23 and onwards).